backend/src/sms/routes/sms-transactional.routes.ts

```diff
@@ -0,0 +1,61 @@
+import { Router } from 'express'
+import { celebrate, Joi, Segments } from 'celebrate'
+import { SmsMiddleware, SmsTransactionalMiddleware } from '@sms/middlewares'
+
+const router = Router({ mergeParams: true })
+
+// Validators
+const sendValidator = {
+  [Segments.BODY]: {
+    body: Joi.string().required(),
+    recipient: Joi.string().trim().required(),
+    label: Joi.string().required(),
+  },
+}
+
+// Routes
+
+/**
+ * @swagger
+ * path:
+ *   /transactional/sms/send:
+ *     post:
+ *       summary: "Send a transactional SMS"
+ *       tags:
+ *         - SMS
+ *       requestBody:
+ *         content:
+ *           application/json:
+ *             schema:
+ *               required:
+ *                 - body
+ *                 - recipient
+ *                 - label
+ *               properties:
+ *                 body:
+ *                   type: string
+ *                 recipient:
+ *                   type: string
+ *                 label:
+ *                   type: string
+ *       responses:
+ *         "202":
+ *           description: Accepted. The message is being sent.
+ *         "400":
+ *           description: Bad Request
+ *         "401":
+ *           description: Unauthorized
+ *         "429":
+ *           description: Too many requests. Please try again later.
+ *         "500":
+ *           description: Internal Server Error
+ */
+router.post(
+  '/send',
+  celebrate(sendValidator),
+  SmsMiddleware.getCredentialsFromLabel,
+  SmsTransactionalMiddleware.rateLimit,
```

Comment:
I was thinking, do we actually need to rate limit SMS explicitly? Can we just forward Twilio's rate limit response to the user directly?

Reply:
I think we can. We can throw a specific RateLimitError for the relevant Twilio error codes. I believe they are 20429 [1] and 21611 [2].

[1]: https://www.twilio.com/docs/api/errors/20429
[2]: https://www.twilio.com/docs/api/errors/21611

Reply:
Cool. I think we should do that. Given that there are actually [two mechanisms](https://www.twilio.com/blog/programmable-sms-api-basics-best-practices) at play, API rate limit and message queue. It would be easier to just forward the rate limit response back to the user than to try to replicate that on our side.

Reply:
Resolved in https://github.com/opengovsg/postmangovsg/pull/1017/commits/5d10738f9cdbf87a8e62b9b413d645d7772acf9f